sse2neon.h

```diff
@@ -3624,7 +3624,11 @@ FORCE_INLINE __m128i _mm_minpos_epu16(__m128i a)
 FORCE_INLINE int _mm_popcnt_u32(unsigned int a)
 {
 #if defined(__aarch64__)
+#if defined(__GNUC__)
+    return __builtin_popcount(a);
```

Comment:
Can we assume that both gcc and clang produce the same instruction sequence with `__builtin_popcount`? If so, the check for gcc can be eliminated when target is Aarch64.

Reply:
It was written for the non-supported compilers mostly -- better portability

And yes, both gcc and clang produce at least the code written before